apps/roam/src/utils/triplesToBlocks.ts

```diff
@@ -25,6 +26,12 @@ const triplesToBlocks =
     }[],
   ) =>
   () => {
+    triples.forEach((t) => {
+      posthog.capture("New Relation: Created", {
+        relation: t.relation,
+      });
+    });
+
```

Comment:
@mdroidian Is this code related to "Discourse Relation created" or "Instantiated Discourse Relation"?

Reply:
Neither, specifically.

Technically it is related to "Instantiated Discourse Relation" via the Canvas.
But it is also used in the RelationConfig to create previews.
And could be used elsewhere in the future for something unrelated.

![image](https://github.com/user-attachments/assets/967c91b5-3b79-4cc4-956f-0a9f98e4dfb1)

So `posthog.caputre` would create false positives

Reply:
got it, can you point me to where I can find code relate to "Discourse Relation created"? I was not able to find it, there was a file related to getDiscourseRelation but could not find code related to creation, then I searched for page  and block creation then found this triplesToBlocks and from my naive pov it seemed to be the place where relations were being created.